### DISCUSSION TITLE
Beta Projects should have filters available

### DISCUSSION THREAD

**Discussion by christine-clark** [community/community#15632] [Projects and Issues]
Projects is nice to easily allow you to add custom fields but there is no filtering available. If you need to filter the table view down then you should have an easy way to setup filter options for your custom fields.

**Comment by aquint-zama** [comment]
Filtering is already available, you have a text entry just below your views tabs that let you enter any filter (with autocompletion). You could even keep your filters by default by saving your current view once your filter is active. Hope it helps

<img width="390" alt="Capture d’écran 2022-04-28 à 11 12 36" src="https://user-images.githubusercontent.com/86411313/165719464-ef49f02f-8091-4762-a8cb-cb482e854a9e.png">
